sdk/cognitiveservices/cognitiveservices-qnamaker-runtime/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
can we inherit from the root tsconfig instead?

Reply:
Do you mean inheriting from azure-sdk-for-js/tsconfig.json? Is something like this what you're thinkging?
![image](https://user-images.githubusercontent.com/21109913/73476002-515c5a00-4346-11ea-9d0b-624938a603f1.png)

Reply:
There is a package.tsonfig.json in the root that sets module to es6, so you can extend that to get rid of one line. 

We recommend skipping lib too if possible, but I'm not sure if that is compatible with track1 without too much work.

Reply:
We need it there, otherwise we start getting a bunch of errors at build. @daviwil told me the reason, but I forgot. We also had to add "dom" to the lib array

Reply:
Yeah, we're not supposed to force Node devs to use 'dom' (which doesn't exist for them), but it seems that node-fetch is putting that unpleasant requirement upon us for now.